bin/pre_deploy.sh

```diff
@@ -14,6 +14,6 @@ else
 fi
 
 echo "-----> PRE-DEPLOY: Running Django migration..."
-./manage.py migrate --noinput
+newrelic-admin run-program ./manage.py migrate --noinput
```

Comment:
I doubt this. We don't need newrelic to understand the db migrations.

Reply:
👍